src/templates/cda/Resources/Composition.hbs

```diff
@@ -76,7 +76,7 @@
                                     {{/each}}
                                 {{/each}}
                             {{else if resObj.resourceType}}
-                                {{#each ../this.section.entry}}                           	
+                                {{#each (toArray../this.section.entry)}}                           	
```

Comment:
Should there be space between 'toArray' and '..'?

Reply:
OK，I added it!